DevelopedCalculator.js

```diff
@@ -24,7 +24,7 @@ class DevelopedCalculator extends Calculator {
     }
     gettingHistoryArray() {
         this.history.join(",");
-        this.historyElement.textContent = this.currentResult;
+        this.historyElement.textContent = this.history;
```

Comment:
`this.history` is the array of the clicked button,
I don't know what is the point of having this ` this.history.join(",");` if you're not using it?